public/app/features/alerting/unified/hooks/useProduceNewRuleGroup.test.tsx

```diff
@@ -0,0 +1,237 @@
+import { HttpResponse } from 'msw';
+import React from 'react';
+import { render, userEvent } from 'test/test-utils';
+import { byRole, byText } from 'testing-library-selector';
+
+import { setBackendSrv } from '@grafana/runtime';
+import { backendSrv } from 'app/core/services/backend_srv';
+import { CombinedRule } from 'app/types/unified-alerting';
+import { RulerGrafanaRuleDTO } from 'app/types/unified-alerting-dto';
+
+import server, { setupMswServer } from '../mockApi';
```

Comment:
(Bit of an inconsistency on my part, but you can just use the return value from the `setupMswServer` method, you don't _have_ to import `server`, but it is clearer that way 🤷 )

Reply:
Oh sounds more reasonable to use the return value here :D